gradle.properties-example

```diff
@@ -0,0 +1 @@
+android.useAndroidX=true
```

Comment:
Why do we use a `-example` file and copy it on every step of the pipeline?

Reply:
`gradle.properties` file is currently [ignored](https://github.com/tumblr/PermissMe/blob/master/.gitignore#L11).

Using `gradle.properties` to pass credentials and then ignoring it in the project is a common pattern and since it was an existing ignore I didn't want to change it. We use the same pattern in `wordpress-mobile` projects and we provide `gradle.properties-example` file in the projects that has the necessary properties. For example, [here is the one for WordPress-Android](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/gradle.properties-example) and the [copy instruction](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/.buildkite/pipeline.yml#L12).

I don't love this pattern and I think we should use a `secrets.properties` file to hold any sensitive data, but because `gradle.properties` is also used for user specific settings (such as memory) it's a big commitment to change the way this is handled.

I also think it was a mistake by Google to require the `useAndroidX` to be passed as a Gradle property and should have just been another property in the `android` buildscript block. This would have made it so that this project doesn't need a `gradle.properties` file. 🤷

Reply:
TIL thanks for the context 👍 

>I also think it was a mistake by Google to require the useAndroidX to be passed as a Gradle property and should have just been another property in the android buildscript block

yea totally agree